app/components/candidate_interface/application_dashboard_course_choices_component.rb

```diff
@@ -52,15 +52,6 @@ def show_missing_banner?
       @show_incomplete && !@application_form.course_choices_completed && @editable
     end
 
-    def course_change_path(application_choice)
-      if multiple_courses?(application_choice)
-        candidate_interface_course_choices_course_path(
-          application_choice.current_course.provider.id,
-          course_choice_id: application_choice.id,
-        )
-      end
-    end
```

Comment:
Not possible to change courses from 2023 applications (and this method is not used anywhere)